### ISSUE TITLE
Document developer usage experiences on Windows 10 with WSL and Docker

### ISSUE THREAD

**Issue by jornh** [gtoonstra/databook#8]
Right now https://github.com/gtoonstra/databook#prerequisites clearly states:

> "You'll need a Mac or Linux with a docker installation to run the sample deployment of databook."

I don't own a Mac, and I'd like to use a combination of [WSL](https://docs.microsoft.com/en-gb/windows/wsl/about) (a Ubuntu based sub-system that keeps getting closer and closer to being a "capable enough" Linux) and Docker on my Windows machine in place of having to wrangle a VM.

This issue (which I guess I just volunteered to work on fixing 😉) is just to have a place to track what I come across: 

- [x] Today I just got one step closer to it working, as company IT here finally let Win 10 version 1709 (Fall Creators Update) out of the bag. It resolved an issue with gUnicorn couldn't run because WSL was missing `/proc/<pid>/status` in Win 10 version 1703 (this BTW also affected `airflow webserver` with Airflow 1.9.0)

   So, long story short: To run `databook webserver` on WSL bash - a prerequisite is you need to be on `v1709`

- [ ] Windows docker uses windows mount point names. This doesn't work well with the current docker-compose files. I hope https://nickjanetakis.com/blog/setting-up-docker-for-windows-and-wsl-to-work-flawlessly#ensure-volume-mounts-work will provide an elegant fix to that

- [ ] more...? I hope not ...

When I have it working my plan is to submit a PR on the README or something ...
